system/lib/standalone/standalone.c

```diff
@@ -333,3 +333,15 @@ weak char* _emscripten_sanitizer_get_option(const char* name) {
 weak char* emscripten_get_module_name(char* buf, size_t length) {
   return strncpy(buf, "<unknown>", length);
 }
+
+// Stubs for non-busy sleep
+int
```

Comment:
Can you make these `weak` like all the above functions (and put the return code on the same line)?